### ISSUE TITLE
Wizard101 Support?

### ISSUE THREAD

**Issue by Lasercar** [niftools/blender_niftools_addon#571]
**Is your feature request related to a problem? Please describe.**
Niftools can't properly import the Gamebryo file format versions used by Wizard101. For example, it doesn't recognise NiPixelData used for nif/embedded textures (though NifSkope does though) or a NiPathInterpolator used in one scene I tried importing.

**Describe the solution you'd like**
-

**Describe alternatives you've considered**
-

**Additional context**
To get/reach the nif files, use quickbms with the Wizard101 Kiward script.

**Comment by Candoran2** [comment]
Could you attach an example of such a nif to this issue?

**Comment by Lasercar** [comment]
Sure. Do you want me to upload one of the archives or just one of the nifs? I don't they keep their textures mostly separate.

**Comment by Candoran2** [comment]
One of the nifs, preferably with the associated textures.

**Comment by Lasercar** [comment]
Hmm, ok. Try this one.
[MB_Hub.zip](https://github.com/niftools/blender_niftools_addon/files/11634755/MB_Hub.zip)

**Comment by Lasercar** [comment]
And, the textures.
[Textures.zip](https://github.com/niftools/blender_niftools_addon/files/11634803/Textures.zip)

**Comment by Candoran2** [comment]
Thanks for the information. It looks like the NiPathInterpolator issue is non-trivial to solve, so it might take a while ere that is addressed. As for the NiPixelData, textures packed _within_ a nif file usually get created as dds files upon import - the ones in your example nif are used in a NiFlipController, so maybe that's why it's not happening. For the ones existing externally as a nif, we'd either have to make Blender be able to load nif files as textures, or convert every texture nif file to a dds/other format when importing - both of those are also complicated.

All that is to say - it might be a while before these features are implemented.

That being said, if you remove the NiPathController in your example nif, you can at least import the geometry without issues:
![image](https://github.com/niftools/blender_niftools_addon/assets/45334438/064857d9-cf93-4ce0-b219-b3c629792f47)

**Comment by EthanZeigler** [comment]
Is there any sort of timeline for this? Not asking for more than a general guess, even if it's "IDK", but I'm running into the same issue.

**Comment by r-best** [comment]
I was looking into this myself today, the Blender plugin can't import the textures properly but NifSkope can; if you import a NIF into NifSkope and manually export the textures for each object as `.tga` files, you can save all those TGAs on your computer and reassign them as image textures in Blender
It's a very manual process but it seems to work based on a simple test, I was able to import the Marleybone player house exterior NIF into Blender via this plugin and then separately export `WC_Door02_JT.tga` for the world door in NifSkope, and then reassign the door's texture in Blender to the TGA:

tldr there's a workaround but it's a pretty involved manual process

![image](https://github.com/niftools/blender_niftools_addon/assets/19865935/a307ce4c-9e66-4d88-bd0f-327073f20d35)

**Comment by EthanZeigler** [comment]
That aligns with my experience. it may be possible to automate the process but it would be both cumbersome and break exporting them again.
